OoyalaAPISampleApp/app/src/main/java/com/ooyala/sample/players/BasicPlaybackVideoPlayerActivity.java

```diff
@@ -100,6 +105,11 @@ public void update(Observable arg0, Object argN) {
       return;
     }
 
+    // Automation Hook: to write Notifications to a temporary file on the device/emulator
+    String text="Notification Received: " + arg1 + " - state: " + player.getState();
+    // Automation Hook: Write the event text along with event count to log file in sdcard if the log file exists
+    playbacklog.writeToSdcardLog(text);
+
     Log.d(TAG, "Notification Received: " + arg1 + " - state: " + player.getState());
```

Comment:
I think the text variable can be reused for the Log here too. Please use it as `Log.d(TAG, text);`